examples/falcon_discover_hosts.rs

```diff
@@ -33,9 +33,12 @@ async fn main() {
     let mut offset = 0usize;
```

Comment:
(non-blocking comment) I see that the second param for `query_host` is `i32` so I wonder if the `offset` should ever be `usize`? If it shouldn't then should the code panic if `response.resources.len()` OR `offset + response.resources.len()` cannot fit into `i32`?

Reply:
I'm wondering if we can simply the declaration of offset to this?

`let mut offset = 0;`

Reply:
I think the APIs types are incorrect (well, not necessarily incorrect, but we didn't look at the definitions), I believe what happened here is that the API spec says `int` which is interpreted by the auto generator as the smallest standard `int` type which would be an `int32`. 
I guess it doesn't make sense for neither `limit` nor `offset` to be allowed to be a negative number.

I suggest we look at the API definitions as soon as we get the latest spec converted and pushed to main.